feat(site_header): Add aria label to language switcher - FRONT-3658

emeryro opened the pull request.

papegaill reviewed (approved):
breaking change tag?

emeryro commented:
@papegaill there should be no breaking change here, I just added an extra parameter.
I also fixed the documentation which was wrongly describing a "name" parameter, while it was in fact already called "label" in the code
